subprojects/performance/src/performanceTest/groovy/org/gradle/performance/regression/java/JavaUpToDatePerformanceTest.groovy

```diff
@@ -69,9 +62,14 @@ class JavaUpToDatePerformanceTest extends AbstractCrossVersionPerformanceTest {
         result.assertCurrentVersionHasNotRegressed()
 
         where:
-        testProject                   | parallel
-        LARGE_MONOLITHIC_JAVA_PROJECT | false
-        LARGE_JAVA_MULTI_PROJECT      | true
-        LARGE_JAVA_MULTI_PROJECT      | false
+        parallel << [true, false]
+    }
+
+    private JavaTestProject getJavaTestProject() {
+        def javaTestProject = JavaTestProject.values().find { it.projectName == runner.testProject }
+        if (javaTestProject == null) {
+            throw new IllegalArgumentException("Cannot find Java test project for ${runner.testProject}")
+        }
+        return javaTestProject
     }
```

Comment:
Hm, why is this code here? Won't it be used for every performance test?

Reply:
Yeah, this needs to go somewhere else. Currently, this is the only performance test converted to the new infrastructure, this is why I left it here.

Reply:
This is now in `TestProjects`.